### PR TITLE
Use automation label in device list for devices accessed using API

### PR DESCRIPTION
**Pull request by vbanthia**
User will see `Stop Automation` label for devices accessed using API instead of `Stop Using` 

screen shot 2016-11-23 at 6 42 47 am

### CODE REVIEW

**Comment by gunta** [comment]
LGTM. 
If it turns out to be still confusing, it might be good to create a new css style for `state-automation`.

**Comment by vbanthia** [comment]
Color for `using` state & `automation` state are different. `using` has `btn-primary` while `automation` has `btn-info`. So, I don't think there will be any confusion. Could not compare both button in screenshot because I only one device now.

File: res/app/components/stf/keys/add-adb-key/add-adb-key.pug [hunk 1]
```diff
@@ -36,4 +36,7 @@
         i.fa.fa-plus.fa-fw
         span(translate) Add Key
 
+      br
```
Comment (sorccu) [inline res/app/components/stf/keys/add-adb-key/add-adb-key.pug]: Not sure why these are necessary

**Comment by sorccu** [comment]
Some seemingly unnecessary old-school `<br>`s in one of the templates, other than that LGTM.

File: res/app/components/stf/keys/add-adb-key/add-adb-key.pug (hunk 1 above)
Comment (vbanthia) [inline res/app/components/stf/keys/add-adb-key/add-adb-key.pug]: Yeah, I forgot to remove these. Need to add some margin before error-message alert box.

**Pull request merged**